doc/plugin_tutorial/tuto2/src/persistent_counter.ml

```diff
@@ -0,0 +1,57 @@
+(*
+ * This file defines our persistent counter, which we use in the
+ * CountPersistent command.
+ *)
+
+(*
+ * As before, we want to override references:
+ *)
+open Summary.Local
```

Comment:
I think usually we don't open Summary, we fully qualify instead.

Reply:
Hm, but then how do you get the syntax, like `:=`?

Reply:
Summary.ref generates a standard "ref" afaict, and the := from OCaml is just fine.

In the special case of Summary.Local, I'd open the module locally as in `let open ... in` since it would otherwise mask the standard := for the whole file and use Summary.Local.ref to declare the local_ref.